test/unit/lib/cli/interactive-setup/aws-credentials.test.js

```diff
@@ -348,10 +348,10 @@ describe('test/unit/lib/cli/interactive-setup/aws-credentials.test.js', () => {
       expect(context.stepHistory.valuesMap()).to.deep.equal(
         new Map([
           ['credentialsSetupChoice', '_local_'],
-          ['createAwsAccountPrompt', true],
-          ['generateAwsCredsPrompt', true],
-          ['accessKeyId', '_user_provided_'],
-          ['secretAccessKey', '_user_provided_'],
+          ['createAwsAccountPrompt', '_user_input_'],
+          ['generateAwsCredsPrompt', '_user_input_'],
```

Comment:
This looks confusing. We dedicated `_user_input_` to user's free form text input, while here is definitely not a case.

Can't we resolve it to something as `_continuation_` ?

Reply:
I guess we can - the actual answer that is being record is empty string, so we could default to `_continuation_` in such cases or just record an empty string - what do you think would be better. I like the `_continuation_` suggestion

Reply:
Yes, I'd stick to `_continuation_` (nothing better comes to my mind)